AjobK/Athena
# 203 view liked posts & 198 view liked posts on profile

closes #203
closed #198

 **What is it supposed to do**
Everyone can see who liked a post and can see what posts a user liked on their profile (yes I put two different use cases in one branch)

 **How can we test your branch**
- Go to a post and click on the amount of a like to open the list (shouldn't be possible if there are no likes)
- Click on a profile in the list, test whether it's sorting properly or whatever
- Go to a profile and look at the liked posts. See if this updates properly and is sorted.

 **What broweser have been checked**
